packages/react/src/components/Authenticator/VerifyUser/VerifyUser.tsx

```diff
@@ -39,7 +39,7 @@ const generateRadioGroup = (
           name="unverifiedAttr"
           value={key}
           key={key}
-          defaultChecked={index === 0}
```

Comment:
`defaultChecked` is not used to pass in an initial boolean value of a radio button, [it is to return the initial boolean value of the radio button](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/defaultChecked). Setting it here will not actually set the value of the radio button.

Reply:
This actually looks like it isn't the case. It appears that in the Amplify Radio component, `defaultChecked` does indeed set the default state of the checkbox